modules/EVSE/EvseManager/EvseManager.cpp

```diff
@@ -1694,6 +1702,24 @@ bool EvseManager::cable_check_should_exit() {
     return charger->get_current_state() not_eq Charger::EvseState::PrepareCharging;
 }
 
+bool EvseManager::check_voltage_to_protective_earth_in_range(types::isolation_monitor::IsolationMeasurement m) {
+    static const double MAX_VOLTAGE_STATIC = 550.0; // defined by IEC 61851-23:2023, $6.3.1.112.2
+    if (m.voltage_V.has_value() and m.voltage_to_earth_l1e_V.has_value() and m.voltage_to_earth_l2e_V.has_value()) {
+        if (powersupply_capabilities.max_export_voltage_V > 500) {
+            double max_allowed_value = m.voltage_V.value() * 1.1;
+            return (abs(m.voltage_to_earth_l1e_V.value()) < max_allowed_value and
```

Comment:
abs() is poorly defined in C++: it may be int or float. Maybe cleaner to use std::abs() from  cmath